### ISSUE TITLE
[EPIC] Facility Locator Mobile Map experience

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-cms#16001]
## Value to Veterans
Veterans viewing Facility Locator search results on a mobile device will understand the relationship between search results and indicators on the map 

## Headlines
**OCTO owner** - Michelle Middaugh
**Is request tracked to an event or Congressional deadline?** - No

## Details
**Who is the primary user?**
Veterans and beneficiaries

**What are the user goals for the project/page/product?**
Veterans viewing Facility Locator search results on a mobile device will understand the relationship between search results and indicators on the map  and will know to get the information they need about a selected location. 

Will also include updates to make results more accessible, such as
- Using numbers instead of letters for the markers
- Adding the location marker to the heading with the title of the search result
- Moving mileage after the heading for each result
- Displaying TTY links for users who are deaf or hard of hearing per recommendations in ticket

## Top-line Description / Goal(s)
Prepare a staging prototype to enable mobile map useability testing

### In scope
- Interactive map markers
- Add TTY Numbers to Search Results
- Maintain Selection between list and map tabs (if deemed feasible in #17174)
### Out of scope
- "Search this area" blocks too much of the map
- Differentiating between location indicators that are very close

**Comment by mmiddaugh** [comment]
**Updated 12.11.2023**
- This work has been prioritized for early 2024 but is dependent on FE capacity to build new experience in Staging to support usability testing with Veterans who use assistive tech

**Comment by mmiddaugh** [comment]
See also [Visual indication of all the results in the list of the Facility Locator](https://app.zenhub.com/workspaces/sitewide-facilities-639f5253e4b702a32376339e/issues/gh/department-of-veterans-affairs/va.gov-cms/10083)

**Comment by mmiddaugh** [comment]
**Update 2.7.2024**
Research planned but waiting on engineering capacity to build prototype

**Comment by mmiddaugh** [comment]
Update 3.8.2024
- Research planned but waiting on engineering capacity to build prototype
- Returning to backlog

**Comment by jilladams** [comment]
For our collective memories: we're regression testing the Facility Locator with respect to the LIghthouse V1 API update. For that, I made a regression testing workbook that is UI-test heavy: https://docs.google.com/spreadsheets/d/1wgIM4oIwAQk-E151z07P3hhZuQJ6mhepv7U5tZjdpEI/edit?usp=sharing. This is a base we can use to think about testing the mobile map iteration, when we get there.

**Comment by mmiddaugh** [comment]
**Update 6.7.2024**
- Still on hold pending engineering capacity

**Comment by mmiddaugh** [comment]
**Update 8.8.2024**
no change

**Comment by mmiddaugh** [comment]
**Update 9.3.2024**
Beginning small enhancements now as steps toward build for mobile map in Q42024 (calendar year)

**Comment by mmiddaugh** [comment]
**Update 11.5.2024**
- this work is a prioritized initiative for 4Q2024 (CY)

**Comment by jilladams** [comment]
From story mapping today: 
* We refined / closed old Facility Locator epics, and organized all remaining mobile map issues into this epic.
* @Agile6MSkinner has a list of tasks, some are already ticketed, I think a couple still need new tickets. (In MIchael's internal A6 mural)
* The existing tickets in this epic are tagged correctly and queued to refine.

**Comment by Agile6MSkinner** [comment]
We will need to use findings from #20269 when deploying to staging.

**Comment by mmiddaugh** [comment]
**Update 2.14.2025**

- Engineering nearly complete
- Research in planning

**Comment by jilladams** [comment]
From UX sync: Given: 
* Perigean recruitment changes
* End of PoP upcoming 
* Potential risk to shipping both Mobile Map and Progressive Disclosure changes , with both of those in mind

Some strategic changes to our plans for these FL features.

**Mobile map / Progressive Disclosure**: 
want to move ahead toward trying to launch, and not block on research. **Next step: staging review.** Scheduling will require
    * All the artifacts (QA standards, Technical architecture, Foundational a11y testing, etc etc etc etc, check the Collab Cycle docs)
    * 4-business-day lead time for Governance to review those artifacts before Staging Review date

**Services Autosuggest:** 
* will still pursue research options, Brittany will continue on plan / convo guide as soon as mobile map artifacts are settled. 
* Study will still include Progressive Disclosure since it's a desktop/mobile research plan, just the feature will be live in prod (assuming we get to ship) while we research.

**Comment by jilladams** [comment]
Research sessions from 4/1 forward have been canceled. 

Feature flipper was enabled in production this morning at ~10:15am PT.
